lib/git.rb

```diff
@@ -376,8 +376,8 @@ def fetch(remote = nil, options = {})
       fetch_command.set_thin(true) if options[:thin]
       fetch_command.set_check_fetched_objects(true) if options[:check_fetched_objects]
       fetch_command.set_remove_deleted_refs(true) if options[:remove_deleted_refs]
-      fetch_command.set_ref_specs(RefSpec.new(options[:refspecs])) if options[:refspecs]
       fetch_command.set_remote(remote) if remote
+      fetch_command.set_ref_specs(options[:refspecs]) if options[:refspecs]
```

Comment:
Apparently on newer JDK's `.set_ref_specs(RefSpec.new('<refspec>'))` was no longer equivalent to `.set_ref_specs('<refspec>')`